src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Ogooreck.Sample.BusinessLogic.FSharp.Tests.fsproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+
+        <IsPackable>false</IsPackable>
+        <GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
pretty sure not required either

Reply:
Yup, when I removed Program.fs it's not required anymore.